### ISSUE TITLE
Builds no longer work

### ISSUE THREAD

**Issue by knightinlet** [pimoroni/pico-boilerplate#8]
Since commit 0c34de23f53ec23ea852560d873fbce65a09d949 of Pimoroni-pico (Tidy up include paths) you now experience include build failures when using the boiler plate template.

**Comment by Gadgetoid** [comment]
Good catch, thank you! I'm working on a fix.

**Comment by Gadgetoid** [comment]
Should be fixed by #9

**Comment by knightinlet** [comment]
Thanks Phil for investigating very much appreciated.  To get it to work I had to copy the pimoroni_pico_import.cmake file to the pimoroni_pico directory as the make file checks for its existence. Is this correct or will pimoroni_pico_import.cmake be distributed with pimoroni_pico in a future release. Thanks again

**Comment by knightinlet** [comment]
Apologies if it's my lack of knowledge. Whenever I referenced one of the breakouts such as bme68x or rv3208 it would always complain it could not link to pimoroni_i2c. The way I got it to work was to clone the common cmakelists.txt and create a common.cmake file within the common directory. I then just included it as I do for any of the breakout boards in my project and everything was then happy. Just wondered if it was just me. Thanks.
